content/exchange/artifacts/Linux.Forensics.Targets.yaml

```diff
@@ -0,0 +1,37 @@
+name: Linux.Forensics.Targets
+author: Cedric MAURUGEON - @kidrek
+description: |
+  This artifact collects all necessary artifacts files and directories from Linux operating system. 
+  
+reference:
+  - https://github.com/ForensicArtifacts/artifacts/blob/main/artifacts/data/linux.yaml
+
+parameters:
+  - name: BootTargets
+    default: /boot/grub/grub.cfg,/boot/grub2/grub.cfg,/boot/initramfs*,/boot/initrd*,/etc/init.d/**,/etc/insserv.conf,/etc/insserv.conf.d/**
```

Comment:
it is probably better to use parameters of type csv - then the GUI will render them nicely in a table and also you can add the globs below because they will be just arrays - so you dont need to join with "," and split them again.

Reply:
Thx for this advice.
I'll integrate it into the next release

Reply:
I wrote the new version of parameters using csv type, but i stuck on the query ```SELECT OSPath, upload(file=OSPath) AS Upload```. Could you help me to concatenate globs, please ?

Reply:
I modified parameters ;)